src/app/components/typography.tsx

```diff
@@ -19,50 +19,42 @@ const openSauceMetrics = {
 const h1 = {
```

Comment:
I got rid of `leading` as it's not a valid CSS prop and I noticed it when investigating this. 

![Screenshot 2023-06-28 at 09 55 22](https://github.com/hirosystems/wallet/assets/2938440/77be5f96-7fe3-40aa-89bd-51fc68e57d50)

`leading` and `line-height` are usually the same in design terms but if I swap to `lineHeight` here it really effects the UI so I think this is un-needed code. 

I may have missed something though so I can drop this commit if needed